jest.test.config.js

```diff
@@ -12,9 +14,7 @@ module.exports = {
     // (e.g. with babel).
     // However we need to instruct jest to actually transpile our local shared
     // dependencies (e.g. `packages`) which are symlinked as dependencies
-    // in node_modules. Since the name of those packages are scoped with
-    // `@commercetools-local` we simply match the JS files within those folders.
-    'node_modules/@commercetools-frontend/ui-kit',
```

Comment:
I'm not sure why we had this but it shouldn't be needed

Reply:
Oh now I remember 🤦‍♂️
It's the only dependency out of `@commercetools-frontend` that does not need to be transpiled...